logging/README.md

```diff
@@ -32,7 +32,7 @@ Before deploying, you must perform these tasks:
 - [create a local copy of the repository](#log_loc_copy)
 - [customize your deployment](#log_custom)
 
-Note: If you are deploying in a highly secure environment (in which pods cannot connect to the internet and network traffic is routed through a proxy server), you must add `localhost` to the list of hosts that are not routed through the proxy server. If you do not, the Elasticsearch deployment fails.  
+Note: If you are deploying in a highly secure environment (in which pods cannot connect to the internet and network traffic is routed through a proxy server), ensure that `localhost` is included in the list of hosts that are not routed through the proxy server. If it is not included, the Elasticsearch deployment fails, because the deployment scripts use port forwarding to make REST API calls to Elasticsearch and Kibana.  
```

Comment:
I like the revisions.  But, thinking about this further I realized pod connectivity is really not relevant here.  The issue we're addressing relates to the network configuration of the machine on which the deployment script is running rather than any of the pods.  In fact, the deployment scripts could be (likely are) run on a machine outside of the K8s cluster completely.  Therefore, the first sentence should be revised to clarify that if the deployment scripts are being run on a machine which has its network traffic routed through a proxy server, the proxy settings need to include "localhost" in the list of "acceptable" hosts.